lib/airbrake.js

```diff
@@ -54,10 +55,11 @@ Airbrake.PACKAGE = (function() {
   return JSON.parse(json);
 })();
 
-Airbrake.createClient = function(key, env) {
+Airbrake.createClient = function(key, projectId, env) {
```

Comment:
Yup, since we're providing project ID and API key in that order on the project settings page, it feels more natural to follow that order here as well.